src/Photos/Enums.cs

```diff
@@ -76,6 +76,11 @@ public enum PHVideoRequestOptionsDeliveryMode : long {
 	[Mac (10,12, onlyOn64 : true)]
 	[Native]
 	public enum PHCollectionListType : long {
+		[Introduced (PlatformName.iOS, 8, 0, message: "Will be removed in a future release")]
+		[Deprecated (PlatformName.iOS, 13, 0, message: "Will be removed in a future release")]
```

Comment:
The message is not helpful - since deprecated implies it's not to be used. Also do not copy/paste strings directly from sharpie (since those comes from header files).